### PR TITLE
Webp cover support

### LINKED ISSUE #253: Support webp format for covers
I tried to use an image in webp format as a cover. Once I choose the image from the File portal and Tagger is supposed to handle it, it crashes. I tried different webp images on different songs with the same result.
 When running the app from the terminal, this error message appears:

bash-5.2$ flatpak run org.nickvision.tagger
Unhandled exception. GLib.GException: Format d’image non reconnu
   at GLib.Error.ThrowOnError(ErrorHandle error)
   at Gdk.Texture.NewFromBytes(Bytes bytes)
   at NickvisionTagger.GNOME.Views.MainWindow.SelectedMusicFilesPropertiesChanged() in /run/build/org.nickvision.tagger/NickvisionTagger.GNOME/Views/MainWindow.cs:line 1128
   at GLib.Internal.SourceFuncNotifiedHandler.<.ctor>b__4_0(IntPtr userData)

Suggesting this is indeed the format of the image that causes the problem.

I use Fedora 38.

### PR DESCRIPTION
**Pull request by fsobolev**
Closes #253 

1. In MainWindow `image/*` filter in FileDialog was replaced with jpeg/png/bmp/webp, because filter with wildcard crashes portal in KDE (bad KDE, bad). I hope these 4 formats are enough (and I think it's better to have the list of formats that  we can verify instead of allowing any images).
2. `webp-pixbuf-loader` was added to flatpak build
3. Because in flatpak we install webp loader not to main gdk-pixbuf directory (it's read-only), we have to explicitetly load modules from new directory, so there are changes in `Program.cs`. This doesn't affect other packages, local build in fedora loads `webp-pixbuf-loader` installed with dnf without any problem since it's in main directory.

### CODE REVIEW

**Comment by nlogozzo** [comment]
Can you or @soumyaDghosh update snap file as well?

**Comment by nlogozzo** [comment]
Also, if you could update the changelog :)

**Comment by fsobolev** [comment]
I forgot to test export... Well, I suppose it will work on GNOME, but on KDE it crashes again because `GetFirstAlbumArtMimeType` returns `image/*` for webp. Anything we can do with this?

**Comment by nlogozzo** [comment]
> **EDIT**: Probably time for yet another ATL issue 😅 

If we set the mimetype ourselves will it save? Like if we set it to image/png, inage/jpg etc? I don't remember what the exact issue was....does atldotnet always just return image/*? If so then yes I'd open an issue there to remember specific image types.

**Comment by fsobolev** [comment]
> > **EDIT**: Probably time for yet another ATL issue 😅
> 
> If we set the mimetype ourselves will it save? Like if we set it to image/png, inage/jpg etc? I don't remember what the exact issue was....does atldotnet always just return image/*? If so then yes I'd open an issue there to remember specific image types.

I looked at ATL code, yes, it returns `image/*` when it can't detect image format, and there are no checks for WEBP at all https://github.com/Zeugma440/atldotnet/blob/main/ATL/Utils/ImageUtils.cs#L88

**Comment by soumyaDghosh** [comment]
> Can you or @soumyaDghosh update snap file as well?

Webp libraries are in gnome-sdk and probably supported by ffmpeg-sdk also.

**Comment by fsobolev** [comment]
> Webp libraries are in gnome-sdk and probably supported by ffmpeg-sdk also.

`libwebp` is in flatpak GNOME platform as well, but `webp-pixbuf-loader` was missing. You can check by simply trying to import webp cover - if the app crashes, then loader is missing.

File: NickvisionTagger.GNOME/Program.cs [hunk 1]
```diff
@@ -37,18 +37,19 @@ public Program()
             @"* Added offset field to synchronized lyrics page 
               * Added the ability to import and export LRC files in the synchronized lyrics page
               * Fixed an issue where synchronized lyrics stored in the LRC format were not displayed correctly
+              * Fixed an issue where using WebP image for album art caused crash
```
Comment (nlogozzo) [inline NickvisionTagger.GNOME/Program.cs:40]: should be `caused a crash` and then needs to be added to metainfo xml file as well

**Review by nlogozzo — CHANGES_REQUESTED** [review]

**Comment by nlogozzo** [comment]
Code looks good to me, just waiting for new `atldotnet` release to fix the mimetype.

**Pull request merged**